lib/Controller/ViewController.php

```diff
@@ -119,7 +119,10 @@ public function index():TemplateResponse {
 		$this->initialStateService->provideInitialState('show_tasks', $showTasks);
 		$this->initialStateService->provideInitialState('tasks_enabled', $tasksEnabled);
 		$this->initialStateService->provideInitialState('hide_event_export', $hideEventExport);
-		$this->initialStateService->provideInitialState('force_event_alarm_type', $forceEventAlarmType);
+		// provideInitialState does not accept null as a value
+		if ($forceEventAlarmType !== null) {
+			$this->initialStateService->provideInitialState('force_event_alarm_type', $forceEventAlarmType);
+		}
```

Comment:
$this->initialStateService->provideInitialState('force_event_alarm_type', $forceEventAlarmType ?? false);

Reply:
Basically assume no type is forced if it isn't set

Reply:
If we provide `false` then we need to adjust the front-end. E.g.
https://github.com/nextcloud/calendar/blob/c0e10958ae63fce653feb8aa09ab04797309c093/src/components/Editor/Alarm/AlarmList.vue#L81
will return `false` instead of `DISPLAY`, if `forceEventAlarmType` is `false` instead of `null` (if I am not wrong).